include/expr/Parser.h

```diff
@@ -230,7 +230,8 @@ namespace expr {
     /// expressions.
     static Parser *Create(const std::string Name,
                           const llvm::MemoryBuffer *MB,
-                          ExprBuilder *Builder);
+                          ExprBuilder *Builder,
+			  bool QueryIndependence);
```

Comment:
Something has gone wrong with the formatting there I think. Make sure you clang-format your changes so that you don't have to worry about how you indent stuff.